Code/GraphMol/ForceFieldHelpers/ANI/Builder.cpp

```diff
@@ -0,0 +1,50 @@
+//
+//  Copyright (C) 2020 Manan Goel
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#include <RDGeneral/Invariant.h>
+#include <GraphMol/RDKitBase.h>
+#include <ForceField/ForceField.h>
+#include <ForceField/ANI/AtomicContrib.h>
+#include "Builder.h"
+
+namespace RDKit {
+namespace ANI {
+using namespace ForceFields::ANI;
+namespace Tools {
+void addANIContribs(const ROMol &mol, ForceFields::ForceField *field,
+                    std::string model, int confId) {
+  PRECONDITION(field, "bad ForceField");
+  auto conf = mol.getConformer(confId);
```

Comment:
`conf` doesn't seem to be used here. Would you please remove it from this function if it's not needed?